middlewares/config.go

```diff
@@ -37,7 +37,7 @@ func initApp() {
 		if !config.PrestConf.Debug && config.PrestConf.EnableDefaultJWT {
 			MiddlewareStack = append(
 				MiddlewareStack,
-				JwtMiddleware(config.PrestConf.JWTKey, config.PrestConf.JWTAlgo))
+				JwtMiddleware(config.PrestConf.JWTKey, config.PrestConf.JWTJWKS))
```

Comment:
could be better if this is configurable, so old users can still use that config and you can use this also